packages/gatsby-plugin-feed/src/gatsby-node.js

```diff
@@ -32,6 +32,7 @@ exports.onPostBuild = async ({ graphql, reporter }, pluginOptions) => {
     const { setup, ...locals } = {
       ...options,
       ...feed,
+      site_url: feed.link,
```

Comment:
This is already provided through the `site_url: siteUrl` inside the GraphQL query.

To be fair, the docs don't highlight this requirement enough and an addition to the README would be much appreciated!

As such, please revert this change as it would introduce a breaking change (the `link` is optional and thus with this change it would break the plugin for a lot of people)

Reply:
No, `link` is the the `siteUrl`, see: https://validator.w3.org/feed/docs/rss2.html

> link | The URL to the HTML website corresponding to the channel.